README.md

```diff
@@ -11,11 +11,29 @@ A kit takes care of initializing and forwarding information depending on what yo
 
 Please refer to installation instructions in the core mParticle Apple SDK [README](https://github.com/mParticle/mparticle-apple-sdk#get-the-sdk), or check out our [SDK Documentation](http://docs.mparticle.com/#mobile-sdk-guide) site to learn more.
 
+## Deep-linking
 
-## Create Your Own Integration
+Call the mParticle SDK `checkForDeferredDeepLinkWithCompletionHandler:` method to retrieve the respective information.
 
-Detailed instructions on how to implement your own integration with the mParticle Apple SDK can be found [here](https://github.com/mparticle-integrations/mparticle-apple-integration-example/wiki/Kit-Integration-Development).
+The completionHandler will get passed the linkInfo dictionary with the following data. Use the destinationURL to navigate to your desired location within the app.
 
+```json
+{
+	"destinationURL" : <the destination url>,
+	"clickedURL", <the clicked url>
+}
+
+```
+
+In order for the deeplink to be retrieved properly on app load you need to call the mParticle continueUserActivity to store the clicked url from the userActivity.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray * _Nullable))restorationHandler {
+    [[MParticle sharedInstance] continueUserActivity:userActivity restorationHandler:restorationHandler];
```

Comment:
this is called automatically by our SDK since we proxy the app delegate, so can be left out of docs.